resources/idlharness.js

```diff
@@ -840,9 +840,9 @@ IdlInterface.prototype.test_self = function()
         var desc = Object.getOwnPropertyDescriptor(self, this.name);
         assert_false("get" in desc, "self's property " + format_value(this.name) + " has getter");
         assert_false("set" in desc, "self's property " + format_value(this.name) + " has setter");
-        assert_true(desc.writable, "self's property " + format_value(this.name) + " is not writable");
+        assert_true(desc.writable, "self's property " + format_value(this.name) + " is writable");
         assert_false(desc.enumerable, "self's property " + format_value(this.name) + " is enumerable");
-        assert_true(desc.configurable, "self's property " + format_value(this.name) + " is not configurable");
+        assert_true(desc.configurable, "self's property " + format_value(this.name) + " is configurable");
```

Comment:
Instead of a factual statement you could phrase it as a requirement: "X should not be Y" or "X should be Y". That makes it more self-evident that it's currently wrong.